config-path.js

```diff
@@ -21,6 +22,10 @@ function linux () {
 }
 
 module.exports = function (platform) {
+  if (!userHome) {
+    return os.tmpdir();
```

Comment:
Os.tmpdir is already used in index, all I did is changed the location. Looks like it won't though: https://github.com/nodejs/node/blob/master/lib/os.js#L98